oak/server/rust/oak_runtime/src/node/external.rs

```diff
@@ -49,13 +48,12 @@ impl PseudoNode {
             config_name: config_name.to_string(),
             runtime,
             reader,
-            thread_handle: None,
         }
     }
 }
 
 impl super::Node for PseudoNode {
-    fn start(&mut self) -> Result<(), OakStatus> {
+    fn start(self: Box<Self>) -> Result<JoinHandle<()>, OakStatus> {
```

Comment:
Is it possible to refactor the closure inside the `spawn` here to a `worker_thread` like the rest of the PR?

Reply:
Done.